stats/stats.go

```diff
@@ -98,7 +101,9 @@ func (s *InPayload) IsClient() bool { return s.Client }
 
 func (s *InPayload) isRPCStats() {}
 
-// InHeader contains stats when a header is received.
+// InHeader contain stats when the header is received. It is the first event in
+// the server after receiving the RPC. It is followed by the OutPayload
```

Comment:
`OutPayload`

Reply:
Can you clarify what you are asking for here?